src/plugin.rs

```diff
@@ -29,7 +31,7 @@ use crate::{tweenable::ComponentTarget, Animator, AnimatorState, TweenCompleted}
 ///
 /// [`Transform`]: https://docs.rs/bevy/0.12.0/bevy/transform/components/struct.Transform.html
 /// [`Text`]: https://docs.rs/bevy/0.12.0/bevy/text/struct.Text.html
-/// [`Style`]: https://docs.rs/bevy/0.12.0/bevy/ui/struct.Style.html
+/// [`Node`]: https://docs.rs/bevy/0.15.0/bevy/ui/struct.Node.html
 /// [`Sprite`]: https://docs.rs/bevy/0.12.0/bevy/sprite/struct.Sprite.html
 /// [`ColorMaterial`]: https://docs.rs/bevy/0.12.0/bevy/sprite/struct.ColorMaterial.html
```

Comment:
Maybe these links should point to `latest` docs? And also I think `Text` should be swapped for `TextColor` here too.

Reply:
No they shouldn't. If they point to `latest`, then when you look at the docs for version N you get a link to the latest version, which is only N until N+1 is published, and will after that forever be wrong.